bin/fin

```diff
@@ -4372,13 +4373,18 @@ update ()
 	fi
 }
 
-# Return latest fin download url for the latest tag
-get_latest_fin_url ()
+# Return the latest version
+get_docksal_latest_version ()
 {
 	# Get the latest release tag
 	URL_REPO_RELEASE="${GITHUB_API}/repos/docksal/docksal/releases"
-	local latest_tag=$(curl -sSL $URL_REPO_RELEASE | grep tag_name | sed 's/\"tag_name": \"\(.*\)",/\1/' | head -n 1 | tr -d '[:space:]')
+	echo $(curl -sSL $URL_REPO_RELEASE | grep tag_name | sed 's/\"tag_name": \"\(.*\)",/\1/' | head -n 1 | tr -d '[:space:]')
```

Comment:
@achekulaev @lmakarov Does it make more sense to set `DOCKSAL_VERSION` here instead of doing another API call here https://github.com/docksal/docksal/pull/815/files#diff-41adca34c3eac75ca9453c7cb0634f6aR4272